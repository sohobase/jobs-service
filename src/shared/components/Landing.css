@import "../styles/colors.css";
@import "../styles/fonts.css";
@import "../styles/variables.css";

:root {
  --landing-height: 60vh;
}
.landing {
  display: flex;
  align-items: center;
  height: var(--landing-height);
  background-color: var(--background-contrast);

  & .wrapper {
    @apply --wrapper;
  }

  &.slogan {

  }
}
