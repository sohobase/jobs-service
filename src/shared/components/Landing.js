import React from 'react';
import { Link } from 'react-router';
import style from './Landing.css';

export default () => (
  <section className={style.landing}>
    <div className={style.wrapper}>
      <p className={style.slogan}>Lorem ipsum dolor sit amet, consectetur adipisicing elit.</p>
      <input type="search" />
    </div>
  </section>
);
