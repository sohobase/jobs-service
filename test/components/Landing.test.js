import React from 'react';
import renderer from 'react-test-renderer';

import Landing from '../../src/shared/components/Landing';

describe('<Landing />', () => {
  // -- Renders
  it('renders by default', () => {
    const tree = renderer.create(<Landing />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
