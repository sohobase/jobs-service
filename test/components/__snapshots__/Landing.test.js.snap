exports[`<Landing /> renders by default 1`] = `
<section
  className="landing">
  <div
    className="wrapper">
    <h2
      className="slogan">
      Lorem ipsum dolor sit amet, consectetur adipisicing elit.
    </h2>
    <input
      type="search" />
  </div>
</section>
`;
